src/router/paths.ts

```diff
@@ -12,6 +12,11 @@ export enum ProfilePaths {
 	ImportWallet = "/profiles/:profileId/wallets/import",
 	ImportWalletLedger = "/profiles/:profileId/wallets/import/ledger",
 	WalletGroupPage = "/profiles/:profileId/network/:networkId",
+	// Messages
+	SignMessage = "/profiles/:profileId/sign-message",
```

Comment:
@dated is this route along with the `VerifyMessage` route below handled yet? Thinking that if we add them now without handling them it will most probably render an empty screen instead of redirecting to welcome screen :hmhmhm:

Reply:
good catch! yea, those routes will render an empty page currently, but there aren't link from anywhere yet. i'll comment them out for the sake of correctness asap.